22). Add topic files 59 to 62 (#17846)
dotnet/docs: docs/standard/linq/work-dictionaries-linq-xml.md

---
title: How to work with dictionaries using LINQ to XML - LINQ to XML
description: You can convert many kinds of data structures to XML, and you can convert XML to structures. Here is an example that converts a Generic.Dictionary to XML and back.
ms.date: 07/20/2015
dev_langs:
  - "csharp"
  - "vb"
ms.assetid: 57bcefe3-8433-4d3b-935a-511c9bcbdfa8
---

# How to work with dictionaries using LINQ to XML

> It is often convenient to convert data structures of various kinds to XML, and then from XML to other data structures. This article shows a conversion of a <xref:System.Collections.Generic.Dictionary%602> to XML and back.

Reviewer: ```suggestion
It's often convenient to convert data structures of various kinds to XML, and then from XML to other data structures. This article shows a conversion of a <xref:System.Collections.Generic.Dictionary%602> to XML and back.
```
